src/components/button/button-icon.scss

```diff
@@ -11,18 +11,34 @@
   pointer-events: none;
 }
 
-[icon-left] ion-icon {
+[icon-left] ion-icon,
+[dir="ltr"] [icon-left] ion-icon {
   @include button-icon;
 
   padding-right: .3em;
 }
 
-[icon-right] ion-icon {
+[icon-right] ion-icon,
+[dir="ltr"] [icon-right] ion-icon {
   @include button-icon;
 
   padding-left: .4em;
 }
 
+[dir="rtl"] [icon-left] ion-icon {
+  @include button-icon;
+
+  padding-left: .4em;
+  padding-right: initial;
+}
+
+[dir="rtl"] [icon-right] ion-icon {
```

Comment:
That's gonna be a problem.
`icon-left` is for CSS, so when you do:
```
<button ion-button icon-left>
    Button
    <ion-icon name="chatbubbles"></ion-icon>
</button>
```

It doesn't move the icon to the left, just adds some wrong padding.